frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRunJob/PipelineRunJobDetails.tsx

```diff
@@ -0,0 +1,175 @@
+              parameters={pipelineRuntime?.spec.params}
+              taskReferences={taskMap}
+              onClose={() => setSelectedId(null)}
+            />
+          }
+        >
+          <DrawerContentBody>
+            <Drawer isInline isExpanded position="bottom">
+              <DrawerContent
+                panelContent={
+                  <PipelineRunDrawerBottomContent
+                    detailsTab={detailsTab}
+                    onSelectionChange={(selection) => {
+                      setDetailsTab(selection);
+                      setSelectedId(null);
+                    }}
+                    pipelineRunDetails={
+                      job && pipelineRuntime ? { kf: job, kind: pipelineRuntime } : undefined
+                    }
+                  />
+                }
+              >
+                <ApplicationsPage
+                  title={error ? 'Error loading scheduled run' : job?.name}
+                  description={
+                    job ? <MarkdownView conciseDisplay markdown={job.description} /> : ''
+                  }
+                  loaded={loaded}
+                  loadError={error}
+                  breadcrumb={
+                    <Breadcrumb>
+                      {breadcrumbPath}
+                      <BreadcrumbItem isActive>
+                        {error ? 'Scheduled run details' : job?.name ?? 'Loading...'}
+                      </BreadcrumbItem>
+                    </Breadcrumb>
+                  }
+                  headerAction={
+                    loaded &&
+                    !error && (
+                      <PipelineRunJobDetailsActions
+                        job={job ?? undefined}
+                        onDelete={() => setDeleting(true)}
+                      />
+                    )
+                  }
+                  empty={false}
+                >
+                  {nodes.length === 0 ? (
+                    <PipelineTopologyEmpty />
+                  ) : (
+                    <PipelineTopology
+                      nodes={nodes}
+                      selectedIds={selectedId ? [selectedId] : []}
+                      onSelectionChange={(ids) => {
+                        const firstId = ids[0];
+                        if (ids.length === 0) {
+                          setSelectedId(null);
+                        } else if (taskMap[firstId]) {
+                          setDetailsTab(null);
+                          setSelectedId(firstId);
+                        }
+                      }}
+                    />
+                  )}
+                </ApplicationsPage>
+              </DrawerContent>
+            </Drawer>
+          </DrawerContentBody>
+        </DrawerContent>
+      </Drawer>
+
+      <DeletePipelineCoreResourceModal
+        type="scheduled run"
+        toDeleteResources={deleting && job ? [job] : []}
+        onClose={() => {
+          navigate(contextPath ?? `/pipelineRuns/${namespace}`);
+        }}
```

Comment:
Cancelling the delete modal sends the user back to the pipeline run page.
```suggestion
        onClose={(deleteComplete) => {
          if (deleteComplete) {
            navigate(contextPath ?? `/pipelineRuns/${namespace}`);
          } else {
            setDeleting(false);
          }
        }}
```

Reply:
fixed